### PR TITLE
FAQ page

### PR DESCRIPTION
**Pull request by Skejven**
Wiki FAQ page.

___

I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: documentation/src/main/wiki/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# FAQ
+This section contains frequent questions and answers. If you can't find an answer for your question here
```
Comment (RdkLwg) [inline documentation/src/main/wiki/FAQ.md]: > answer for your question
to your question

File: documentation/src/main/wiki/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+please use [Issues Tool](https://github.com/Cognifide/aet/issues) to ask a question.
```
Comment (RdkLwg) [inline documentation/src/main/wiki/FAQ.md]: > to ask a question
to raise a question

File: documentation/src/main/wiki/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+## 1. Setup and first steps
+
+### 1.1. What do I need to star using AET?
```
Comment (RdkLwg) [inline documentation/src/main/wiki/FAQ.md]: > star
start

File: documentation/src/main/wiki/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+- [1. Setup and first steps](#1-setup-and-first-steps)
+  - [1.1. What do I need to star using AET?](#11-what-do-i-need-to-star-using-aet)
+  - [1.2. I have AET environment, how can I run tests?](#12-i-have-aet-environment-how-can-i-run-tests)
+  - [1.3. How can I integrate AET with continuous integration tools?](#13-how-can-i-integrate-aet-with-continuous-integration-tools)
+  - [1.4. I run AET the first time, how the tool will know patterns?](#14-i-run-aet-the-first-time-how-the-tool-will-know-patterns)
+- [2. Environment and software](#2-environment-and-software)
+  - [2.1. Which browser is used by AET?](#21-which-browser-is-used-by-aet)
+  - [2.2. Does AET use database for storing data?](#22-does-aet-use-database-for-storing-data)
+  - [2.3. My page is not english and I have problem with missing fonts. What should I do?](#23-my-page-is-not-english-and-i-have-problem-with-missing-fonts-what-should-i-do)
```
Comment (RdkLwg) [inline documentation/src/main/wiki/FAQ.md]: > english
English

File: documentation/src/main/wiki/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+used by AET can be found in [[Third-party software used by system|SystemComponents#third-party-software-used-by-system]]
+section.
+
+
+### 2.2. Does AET use database for storing data?
+Yes, AET uses [MongoDB](https://www.mongodb.org/) where all data collected during AET suite processing 
+is stored.
+
+
+### 2.3. I have a problem with missing fonts (my page is not english). What should I do?
```
Comment (RdkLwg) [inline documentation/src/main/wiki/FAQ.md]: > english
English

File: documentation/src/main/wiki/FAQ.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
+# FAQ
+This section contains answers for frequently asked questions. If you can't find an answer for your question here
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: _an answer **to** your question_

I think _to_ sounds a lot more natural in this context. Additionally: http://english.stackexchange.com/a/77300/24844

File: documentation/src/main/wiki/FAQ.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+## 1. Setup and first steps
+
+### 1.1. What do I need to star using AET?
+You need Maven and Java installed to run AET and Chrome browser to see reports if you have AET environment
+already set up. See more information in [[Running Suite|RunningSuite#requirements]] section.
+To setup AET you need [VirtualBox 5.1.14](https://www.virtualbox.org/wiki/Downloads) to run AET VM.
+See more details in [[Basic Setup|BasicSetup]] section.
+
+
+### 1.2. I have AET environment, how can I run tests?
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: _I have **an** AET environment_

File: documentation/src/main/wiki/FAQ.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+### 3.6 I want to check how my page behaves on mobile devices, how can I do that?
+AET is not integrated with mobile devices. However, you may simulate mobile resolution and check if your
+page changes its rendering breakpoints for different resolutions. Use [[Resolution Modifier|ResolutionModifier]]
+to simulate different resolutions.
+
+---
+
+## 4. Best practices
+
+### 4.1 I want to check how my page looks on several devices - what is the best way do configure suite?
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: _**a** suite_

File: documentation/src/main/wiki/FAQ.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+```
+The other way is to use [[Login Modifier|LoginModifier]] that can pass simple login form.
+
+
+### 3.4 How can I simulate another user agent?
+Use [[Header Modifier|HeaderModifier]] to achieve that. Just add header modifier with key `'User-Agent'`
+ and requested value.
+
+
+### 3.5 What can I do with elements that render differently each time page is open and it occurs in changes detected each time?
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: The past participle of _open_ is _opened_. Also, you're missing an article.

_each time **a** page is open**ed**_

Notice the difference between the two sentences:

_The door is open._ - in this case, _open_ is an adjective.

_The door was opened._ - in this case, _open_ is a verb. This sentence means that an unspecified party opened the door. This is similar to the context in which you used the word.

File: documentation/src/main/wiki/FAQ.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+as patterns, so the first run will detect no changes. The same is true when a new test or url is added to
+already existing suite - new records will have patterns collected during the first suite run.
+
+
+---
+
+## 2. Environment and software
+
+### 2.1. Which browser is used by AET?
+AET currently uses Fire Fox 38.6.0 ESR as default browser. Information about current versions of components
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: There is no such browser as _Fire Fox_. The name is _Firefox_. It's a single word and, incidentally, a nickname for the [red panda](https://en.wikipedia.org/wiki/Red_panda).

File: documentation/src/main/wiki/FAQ.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+[[SUCCESSFUL BUILD|TrackingProgress#when-tests-successfully-finish---command-line]]
+explore `target` directory and open `redirect.html` in Chrome browser file to see the [[report|SuiteReport]].
+
+
+### 1.3. How can I integrate AET with continuous integration tools?
+AET tests are triggered by [[AET Maven Client Application|ClientApplication]]. Thanks to this you may simply
+run `mvn aet:run` from your favourite CI tool (e.g. [Jenkins](https://jenkins.io/) or 
+[Bamboo](https://www.atlassian.com/software/bamboo)).
+
+### 1.4. I run AET for the first time, how the tool will know patterns?
```
Comment (toniedzwiedz) [inline documentation/src/main/wiki/FAQ.md]: How about: _I'm running AET for the first time. How will the tool know the patterns?_

**Comment by Skejven** [comment]
Thank you for the review guys - fixes provided.

**Pull request merged**